apps/pie-docs/src/components/tag/overview/overview.md

```diff
@@ -0,0 +1,402 @@
+      alt: "A pair of strong and subtle tags in yellow color."
+    } %}
+  {% endcontentItem %}
+  {% contentItem %}
+    <h3>Red</h3>
+    {% contentPageImage {
+      src: "../../../assets/img/components/tag/variation-type-red.svg",
+      width: "106px",
+      alt: "A pair of strong and subtle tags in red color."
+    } %}
+  {% endcontentItem %}
+  {% contentItem %}
+    <h3>Brand</h3>
+    {% contentPageImage {
+      src: "../../../assets/img/components/tag/variation-type-brand.svg",
+      width: "49px",
+      alt: "A subtle tag in brand color."
+    } %}
+  {% endcontentItem %}
+  {% contentItem %}
+    <h3>Neutral - alternative</h3>
+    {% contentPageImage {
+      src: "../../../assets/img/components/tag/variation-type-neutral-alternative.svg",
+      width: "49px",
+      alt: "A subtle alternative for neutral tag."
+    } %}
+  {% endcontentItem %}
+  {% contentItem %}
+    <h3>Outline</h3>
+    {% contentPageImage {
+      src: "../../../assets/img/components/tag/variation-type-outline.svg",
+      width: "49px",
+      alt: "An outline tag."
+    } %}
+  {% endcontentItem %}
+  {% contentItem %}
+    <h3>Ghost</h3>
+    {% contentPageImage {
+      src: "../../../assets/img/components/tag/variation-type-ghost.svg",
+      width: "49px",
+      alt: "A ghost tag."
+    } %}
+  {% endcontentItem %}
+{% endcontentLayout %}
+
+___
+
+## Modifiers
+
+### Emphasis
+
+Tags are available in two types of emphasis. 
+Not all colour available within the subtle emphasis are available within the strong emphasis, for example, Brand, Neutral - alternative, Outline and Ghost have only Subtle variations.
+
+{% contentLayout %}
+  {% contentItem %}
+    <h3>Strong</h3>
+    {% contentPageImage {
+      src: "../../../assets/img/components/tag/modifier-emphasis-strong.svg",
+      width: "49px",
+      alt: "A variation of tag which uses strong colors."
+    } %}
+  {% endcontentItem %}
+  {% contentItem %}
+    <h3>Subtle</h3>
+    {% contentPageImage {
+      src: "../../../assets/img/components/tag/modifier-emphasis-subtle.svg",
+      width: "49px",
+      alt: "A variation of tag which uses subtle colors."
+    } %}
+  {% endcontentItem %}
+{% endcontentLayout %}
+
+### Icon
+
+Icons can be used to visually support the Tag’s label. Only available at the large size.
+
+{% contentLayout %}
+  {% contentItem %}
+    <h3>With icon</h3>
+    {% contentPageImage {
+      src: "../../../assets/img/components/tag/modifier-icon.svg",
+      width: "69px",
+      alt: "A tag with an icon."
+    } %}
+  {% endcontentItem %}
+  {% contentItem %}
+    <h3>Without icon</h3>
+    {% contentPageImage {
+      src: "../../../assets/img/components/tag/modifier-no-icon.svg",
+      width: "49px",
+      alt: "A tag without an icon."
+    } %}
+  {% endcontentItem %}
+{% endcontentLayout %}
+
+--- 
+
+## Sizes
+
+{% contentLayout %}
+  {% contentItem %}
+    <h3>Small</h3>
+    <p>16px height. Icon isn’t available at the small size option.</p>
```

Comment:
Something like this reads a bit more naturally to me.
```suggestion
    <p>16px height. Small tags don't support icons.</p>
```

Reply:
Change made in design